### DISCUSSION TITLE
Backend_Discussions

### DISCUSSION THREAD

**Discussion by ravirajkumarsingh** [ravirajkumarsingh/Covid_HealthCare#3] [Ideas]
For the Backend ideas.

**Comment by AastikSrivastava** [comment]
#Hibernate Dialect 


 ​spring.jpa.properties.hibernate.dialect : org.hibernate.dialect.Oracle10gDialect

**Comment by AastikSrivastava** [comment]
#Application Properties



 
  
 ​spring.datasource.url​ = jdbc:oracle:thin:@localhost:1521:XE 
 ​spring.datasource.username​ = system 
 ​spring.datasource.password​ = bini 
  
 ​spring.jpa.properties.hibernate.show_sql : true 
 ​spring.jpa.properties.hibernate.dialect : org.hibernate.dialect.Oracle10gDialect 
 ​spring.jpa.hibernate.ddl-auto​ = update 
  
 ​management.security.enabled​ = false

**Comment by AastikSrivastava** [comment]
Can we use Rest template in this to consume or use any other project service ?

#Thinking  #just like that
#For Learning purpose

**Comment by ravirajkumarsingh** [comment]
Other rest calls in this api to be use in patients and hospitals Usecase.
